### PR TITLE
Adding Project portfolio section and About section

### PR DESCRIPTION
**Pull request by fitz95**
This Pull request adds a project section grid to the portfolio, which contains a div that holds project details, images, and buttons. Added to this is an about section containing developer information as well as the styling for the mobile view.

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi @fitz95 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

 - [ ] You have submitted a project, that does not have a descriptive PR and a README that does not follow the template provided by Microverse. Kindly refer to this [material](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to get more information on how to set up a professional repository.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by roshan-bajgain — COMMENTED** [review]
Hi @fitz95, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no pull request So, you should have a descriptive title and summary telling you what changes are inside. See the image below for reference

### Pull request title and description
Screenshot 2022-12-03 102024


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏a

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -39,11 +39,133 @@ <h1>Nsong Fitz Gerald</h1>
                 <div class="button">
                     <button id="introbutton">Start collaboration</button>
                 </div>
+            </div> 
+        </section>
+        <section>
```
Comment (bienvenuushindi) [inline index.html:44]: :red_circle: :red_circle: [The project requiremen](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)t states that you should combine both Grid and Flexbox to lay out elements in this section . You used only flexbox. Kindly implement one of the project cards with CSS Grid  :arrow_down: 

requir

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @fitz95 ,

Good job implementing most of the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔


## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- [ ] Please check the inline comments below.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -39,11 +43,135 @@ <h1>Nsong Fitz Gerald</h1>
...
+        </div>
+        <div class="project">
+            <div class="projectimg"><img src="./assets/img/project3.png" alt="project1"></div>
+            <div class="projectdetails">
+                <h3>Project name goes here</h3>
+                <div class="projectbuttons">
+                    <button>HTML/CSS</button>
+                    <button>Ruby on Rails</button>
+                    <button>JavaScript</button>
+                </div>
```
Comment (Gambit142) [inline index.html:93]: - [ ] For semantic reasons, kindly use `ul` tags and `li` tags to list the technology used in each card/project. You can see the example in the first reviewer's comment. https://github.com/fitz95/Portfolio-Microverse-/pull/3/files?file-filters%5B%5D=.css&file-filters%5B%5D=.html&file-filters%5B%5D=.md&show-viewed-files=true#r1038761165

File: style.css [hunk 3]
```diff
@@ -89,3 +89,192 @@ body {
...
+.gridconten h3 {
+  font-size: 20px;
+  font-weight: 300;
+}
+
+#about {
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
```
Comment (Gambit142) [inline style.css:207]: - [ ] Just like the first reviewer stated, kindly align the about with flexbox. Using flexbox helps you to align items properly and it makes your design responsive.

https://github.com/fitz95/Portfolio-Microverse-/pull/3#discussion_r1038762972

Kindly implement this change. 👍

File: index.html [hunk 4]
```diff
@@ -39,11 +43,135 @@ <h1>Nsong Fitz Gerald</h1>
...
+        <section>
+            <div id="titleimg"></div>
+            <div id="projecttitle">
+                <h1 class="title">Projects</h1>
             </div>
-
         </section>
-       
-
+    <section id="projects">
```
Comment (Gambit142) [inline index.html:54]: - [ ] The first reviewer highlighted that you should align elements in the parent container of the project section (`<section id="projects">`) section using grid. 
https://github.com/fitz95/Portfolio-Microverse-/pull/3#discussion_r1038763491

It is a requirement for this milestone. Kindly implement this change. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @fitz95 👋, 

Great work on making the changes requested by a previous reviewer 👏 

 ## Highlights
- All linters checks passed ✔️ 
- Pull request title is descriptive ✔️ 
- Pull request summary is descriptive ✔️
- Included icons in button ✔️ 

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Required Changes ♻️

- [ ] For future projects, kindly use **_imperative_** tense when typing your commit messages. For example, "**_Change_** the code to use hooks" rather than "**_Changed_** the code to use hooks" 👍 

_Kindly check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @fitz95  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Review by Debas-31 — COMMENTED** [review]
Hi @fitz95 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:  :negative_squared_cross_mark:  :negative_squared_cross_mark: 

You have submitted a project with the wrong PR. The project is set up and mobile-first, but you submitted another approved project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mckent05 — APPROVED** [review]
Hi @fitz95,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All linter checks are passing ✔️ 
- You have used the right workflow for this project 👍 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — COMMENTED** [review]
## Hi @fitz95 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted the wrong link. make sure you submit a valid Pull request link that contains the database project, not the portfolio project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by alexander16108** [comment]
Hi @fitz95 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted the wrong link. make sure you submit a valid Pull request link that contains the database project, not the portfolio project.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by christianbiring1 — COMMENTED** [review]
Hi @fitz95 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 
Invalid

You have submitted an invalid PR `LINK`, Kindly make sure you submit the correct link as you may notice this is `Vet clinic database: query multiple tables` but you have submitted the link to the `PORTFOLIO` milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**